api/lib/router/routes/admin/admin-scanner.ts

```diff
@@ -45,15 +45,35 @@ export class AdminScannerController extends ScannerController implements IExpres
       (req, res, next) => this.verifyScannerPermissionsMiddleware(req, res, next, AclOperations.CREATE),
       (req, res, next) => this.addRfidAssignments(req, res, next),
     );
+    app.post(
+      '/scans',
```

Comment:
```suggestion
      '/scan',
```
You are submitting a scan typically, also it will normalize with assign.

Reply:
Done